doc/format.rst

```diff
@@ -753,18 +753,26 @@ monospaced font; in reST, *double* backticks are for ``monospaced font``,
 whereas the behavior of single backticks is defined by the default role. This
 leads to the following style recommendations:
 
-- Module, class, function, method, and attribute names should render as
-  hyperlinks in monospaced font (e.g. :any:`numpy`); depending on project
-  settings, this may be accomplished simply be enclosing them in single
-  backticks. If the hyperlink does not render as intended, explicitly
-  include the appropriate role and/or namespace.
+- Any object that can be linked to should be enclosed in single backticks. That
+  is to say : Module, class, function, method, and attribute names are some of
+  the objects that should be enclosed in single backticks. On a case by cases,
```

Comment:
I think "for example" is more appropriate than "that is" here because it gives examples rather than restating. Also, I assume you meant "on a case-by-case basis"
```suggestion
- For example, module, class, function, method, and attribute names
  should be enclosed in single backticks. On a case-by-case basis,
```

Maybe we do legitimately disagree a little bit. I do think that text that refers to code should always be monospaced. For objects that happen to link, the way to accomplish this is to enclose them in single backticks.

Reply:
Oops, I think I messed up here. I didn't mean to remove the first part: "Any object that can be linked to should be enclosed in single backticks." Otherwise, the change looks like what I had in mind.